src/field_theory/splitting_field.lean

```diff
@@ -200,6 +201,43 @@ begin
   exact splits_of_splits_id _ h
 end
 
+theorem exists_splits (f : polynomial α) :
```

Comment:
This theorem would deserve a docstring. Also, some comments in the proof explaining the strategy and what is going on would be very helpful (but this is already great!)